packages/workspace/package.json

```diff
@@ -63,6 +63,7 @@
   "dependencies": {
     "@parcel/watcher": "2.0.4",
     "chokidar": "^3.5.1",
+    "cli-cursor": "3.1.0",
```

Comment:
This is the final non-ESM version Sindre published, and is the version already in the dependency tree because of our use of `ora` in other packages